packages/inter-protocol/src/auction/scheduleMath.js

```diff
@@ -24,25 +24,48 @@ export const computeRoundTiming = (params, baseTime) => {
   /** @type {NatValue} */
   const lowestRate = params.getLowestRate();
 
+  const noNextAuction = msg => {
+    console.error(assert.error(msg));
```

Comment:
will help for the logs to say what the impact is.
```suggestion
    console.error("noNextAuction:", assert.error(msg));
```

Reply:
done in scheduler.js